prow/jobs/kyma/tests/test-namespace-controller/test-namespace-controller.yaml

```diff
@@ -1,14 +1,14 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
 job_template: &job_template
+  name: kyma-tests-test-namespace-controller
   skip_report: true
-  run_if_changed: "^tests/test-namespace-controller/"
```

Comment:
skip_report: false <- right now we disable Jenkins and make Prow official CI, it is not yet documented ;)